cmd/soroban-rpc/internal/preflight/pool.go

```diff
@@ -49,7 +76,9 @@ func NewPreflightWorkerPool(workerCount uint, jobQueueCapacity uint, ledgerEntry
 func (pwp *PreflightWorkerPool) work() {
 	defer pwp.wg.Done()
 	for request := range pwp.requestChan {
+		pwp.concurrentRequestsMetric.Inc()
```

Comment:
We could do that inside GetPreflight as well, but it might be cleaner to do it here : 
useful metrics for the execution of GetPreflight would be:
1. the time it takes to execute
2. the amount of time spent on DB access for a single GetPreflight
3. successfull vs. non-successful preflights invocations.